DC/5.7.0/digizuite_constants.dcl

```diff
@@ -502,4 +502,10 @@ resource digizuite_config download_destination_id {
     autolink {
         constant = 'DOWNLOADDESTINATIONID'
     }
+}
+
+resource digizuite_config DOWNLOAD_NAME_REPLACE_MASK_FALLBACK {
```

Comment:
We must make the DCL id lowercase. I'm pretty sure that this is required by the parser, and, therefore, you will not even be able to save this. 
Sorry for not noticing this at first, but in the future, please at least test that you can save and autolink when editing the configuration base layers.